internal/services/containerapps/helpers/container_apps.go

```diff
@@ -184,6 +194,54 @@ func ContainerAppIngressSchema() *pluginsdk.Schema {
 
 				"custom_domain": ContainerAppIngressCustomDomainSchemaComputed(),
 
+				"cors_policy": {
```

Comment:
Can we drop `_policy` to stay consistent with other resources implementing `cors` blocks?

```suggestion
				"cors": {
```

Reply:
Fixed.